### PR TITLE
Implemented: app version component from dxp-components(#219)

### LINKED ISSUE #219: Integrate DXP Component: App Version Info on the Settings Page
## What is the motivation for adding/enhancing this feature?
Integrate he DXP component "App Version Info" into the Settings page of the application. This addition will provide users with essential information about the version of the application they are currently using. 

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by Ritika-Patel08**
### Related Issues
 

Related issue #219

 ### Short Description and Why It's Useful
 
Used app version component from the dxp-components in the settings page.

 ### Screenshots of Visual Changes before/after (If There Are Any)
 

 **IMPORTANT NOTICE** - Remember to add changelog entry

 ### Contribution and Currently Important Rules Acceptance
 

 - [x] I read and followed [contribution rules](https://github.com/hotwax/threshold-management#contribution-guideline)

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -94,6 +90,7 @@ import { useRouter } from 'vue-router';
 import TimeZoneModal from '@/views/TimezoneModal.vue';
 import Image from '@/components/Image.vue'
 import { DateTime } from 'luxon';
+import { DxpOmsInstanceNavigator } from '@hotwax/dxp-components';
```
Comment (amansinghbais) [inline src/views/Settings.vue:93]: There is no need to import this component.

File: src/views/Settings.vue [hunk 2]
```diff
@@ -165,9 +157,6 @@ export default defineComponent({
     goToLaunchpad() {
       window.location.href = `${process.env.VUE_APP_LOGIN_URL}`
     },
-    getDateTime(time: any) {
-      return DateTime.fromMillis(time).toLocaleString(DateTime.DATETIME_MED);
```
Comment (amansinghbais) [inline src/views/Settings.vue:169]: Remove the import for DateTime as it is not used.

**Review by amansinghbais — APPROVED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (Ritika-Patel08) [reply src/views/Settings.vue:92]: done.

File: src/views/Settings.vue (hunk 2 above)
Comment (Ritika-Patel08) [reply src/views/Settings.vue:169]: Done.

File: src/views/Settings.vue [hunk 3]
```diff
@@ -93,7 +89,7 @@ import { mapGetters, useStore } from 'vuex';
 import { useRouter } from 'vue-router';
 import TimeZoneModal from '@/views/TimezoneModal.vue';
 import Image from '@/components/Image.vue'
-import { DateTime } from 'luxon';
+import { DxpOmsInstanceNavigator } from '@hotwax/dxp-components';
```
Comment (amansinghbais) [inline src/views/Settings.vue:92]: No need for importing DxpOmsInstanceNavigator

**Review by amansinghbais — APPROVED** [review]

**Pull request merged**